src/devtools/WebAudioGraphIntegrator.ts

```diff
@@ -0,0 +1,213 @@
+import dagre from 'dagre';
```

Comment:
Since this is a new file, can we add 1) a license and 2) a top-level overview?

Reply:
git's diff sadly is not powerful enough to show that WebAudioGraphIntegrator.ts is actually renamed/branched from WebAudioGraphIntegrator.js

Here's a screenshot to give you an slight impression of before and after:
![image](https://user-images.githubusercontent.com/864168/135116434-62edc762-9780-41be-a848-7ccdbd4b51ab.png)

Reply:
I suggest to add license headers for all files at once in a future PR.